5001-6000/LIT5188MashafaMadbal.xml

```diff
@@ -53,22 +53,67 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+          <listRelation>
+            <relation name="saws:hasUsed" active="LIT5188MashafaMadbal#OldTestament"
+              passive="LIT3530OldTes"></relation>
+            <relation name="saws:hasUsed" active="LIT5188MashafaMadbal#PartII"
+                passive="LIT2072NewTes"></relation>
+            <relation name="saws:hasUsed" active="LIT5188MashafaMadbal#FourGospels"
+                    passive="LIT1560Gospel"></relation>
+            <relation name="saws:hasUsed" active="LIT5188MashafaMadbal#Testament"
+                            passive="LIT1938Mashaf"></relation>
+            <relation name="saws:hasUsed" active="LIT5188MashafaMadbal#DoctrinaArcanorum"
+                                    passive="LIT2444Temher"></relation>
+            <relation name="saws:hasUsed" active="LIT5188MashafaMadbal#Draught"
+                                            passive="LIT1560Gospel"></relation>
+          </listRelation>
             <div type="edition">
-                <div type="textpart" subtype="Part 1" xml:lang="gez" xml:id="Introduction">
-                    <label>Introduction</label>
+
+                <div type="textpart" subtype="part" xml:lang="gez" xml:id="PartI">
+                    <label xml:lang="en">Part I</label>
+                    <div type="textpart" subtype="introduction" xml:lang="gez" xml:id="IntroductionI">
+                        <label xml:lang="en">Introduction to the first part</label>
+                    </div>
+                    <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="OldTestament">
+                      <label xml:lang="en">Old Testament and its Apocrypha in verses</label>
+                    </div>
+                    <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="HistoryOfMary">
+                      <label xml:lang="gez">ነገረ፡ ማርያም፡</label>
+                    </div>
                 </div>
-                <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="PartI">
-                    <label>Part I</label>
+
+                <div type="textpart" subtype="part" xml:lang="gez" xml:id="PartII">
+                  <label xml:lang="en">Part II</label>
+                  <div type="textpart" subtype="introduction" xml:lang="gez" xml:id="IntroductionII">
+                    <label xml:lang="en">Introduction to the second part</label>
+                  </div>
+                  <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="Verses">
+                    <label xml:lang="en">Verses addresses to the Holy Trinity</label>
+                  </div>
+                  <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="FourGospels">
+                    <label xml:lang="en">The four Gospels in verses</label>
+                  </div>
+                  <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="Testament">
+                    <label xml:lang="en"><foreign xml:lang="gez">መጽሐፈ፡ ኪዳን፡</foreign>in verses</label>
```

Comment:
I would use here a ref with the work IDs, or otherwise, transcribe the titles for better ease of reading

Reply:
The refs are not appropriate here, as the texts appear to be rather retelling of other texts

Reply:
(It does not matter in this case, but it is my understanding that the refs are used to point to the works in cases like this, they do not state the present of the works)